tests/protocols/test_http.py

```diff
@@ -766,6 +766,7 @@ async def test_invalid_http_request(request_line, protocol_cls, caplog):
     assert b"Invalid HTTP request received." in protocol.transport.buffer
 
 
+@pytest.mark.skipif(HttpToolsProtocol is None, reason="httptools is not installed")
 def test_fragmentation():
```

Comment:
Do we know why this test is only running for `httptools`? 😰

Reply:
because it never happens with h11, https://github.com/encode/uvicorn/pull/1263